src/reducers/index.js

```diff
@@ -289,6 +289,14 @@ export function getSecretsErrorMessage(state) {
   return secretSelectors.getSecretsErrorMessage(state.secrets);
 }
 
+export function createSecretsSuccessMessage(state) {
```

Comment:
naming... selectors should only be retrieving data from the store, so should be `get...` or similar

Reply:
Done 😸